README.md

```diff
@@ -94,6 +94,12 @@ The response to the index will then be:
 Page number is passed through the `page` option of the `render_paginated` method. If none is supplied, `params[:page]` will be used, (or the default parameter configured in the initializer).
 By default, if the page parameter is not present we will use 1 as the page (or the default `page` parameter configured in the initializer).
 The amount of items is passed through the `limit` option of the `render_paginated` method. If none is supplied, `params[:limit]` will be used (or the default parameter configured in the initializer). Default is 25.
+The max amount of items is passed through the `max_limit` option, You can set the value in the initializer or in the `render_paginated` method, (If none is supplied, take the default value configured in the initializer). Default is 50.
```

Comment:
Nice description, but it's only a suggestion :nerd_face: 
What do you think about to move it to the next section `custom options`? 


### Customizing output
#### Custom serializers
A custom serializer for each object can be passed using the `each_serializer` option:
```ruby
render_paginated DummyModel, each_serializer: CustomDummyModelSerializer
```
where the serializer is just an [`ActiveModel::Serializer`](https://github.com/rails-api/active_model_serializers).

#### Custom options
#### max_limit
The max amount of items is passed through the `max_limit` option, You can set the value in the initializer or in the `render_paginated` method, (If none is supplied, take the default value configured in the initializer). Default is 50.

```ruby
render_paginated DummyModel, max_limit: 100
```

##### current_user
Using custom options in serializer, example method `current_user`
```ruby
render_paginated DummyModel, current_user: current_user
```
In serializer
```ruby
class CustomSerializer < ActiveModel::Serializer
  def method
    @instance_options[:current_user]
  end
end
```

Reply:
Is good for me